app/experimenter/nimbus-ui/src/components/PageResults/index.test.tsx

```diff
@@ -128,11 +128,13 @@ jest.mock("../AppLayoutWithExperiment", () => ({
 
     return (
       <div data-testid="PageResults">
-        {props.children({
-          experiment,
-          analysis,
-          refetch: () => Promise.resolve(),
-        })}
+        {typeof props.children === "function"
```

Comment:
It's too bad there isn't a better way to handle this case

Reply:
Well, this case should go away entirely once all the `Page*` components are using the context rather than the functional child pattern

Reply:
Oh wonderful 👍